src/Backups/BackupImpl.cpp

```diff
@@ -287,6 +289,7 @@ void BackupImpl::writeBackupMetadata()
 
     Poco::AutoPtr<Poco::Util::XMLConfiguration> config{new Poco::Util::XMLConfiguration()};
     config->setInt("version", CURRENT_BACKUP_VERSION);
+    config->setBool("deduplicate_files", deduplicate_files);
```

Comment:
It's not necessary, isn't it? I mean this information is not used during `RESTORE`.

Reply:
Yes, it is not used by `RESTORE`. But it will be great to have this information in metadata (I have a script to parse metadata file that can rely on this).